### PR TITLE
Moved shopping card to the end

### LINKED ISSUE #824: Move shopping cart icon to last position in footer
The footer has a shopping cart icon. It should be the last icon in the row
image

### PR DESCRIPTION
**Pull request by Yash282003**
## What type of PR is this? (check all applicable)

- [ ] 🐛 Bug Fix

## Description

This PR fixes the shopping icon position.

## Related Tickets & Documents

Resolves #824 

## Mobile & Desktop Screenshots/Recordings

Screenshot 2023-10-06 213048

## Added code snippets?
- [ ] 🙅 no, because they aren't needed

## Added tests?

- [ ] 🙅 no, because they aren't needed

### No tests? Add a note
because it was a very small fix so not required.

## Added to documentation?

- [ ] 🙅 no documentation needed

### No docs? Add a note
because it was a very small fix so not required.

### CODE REVIEW

File: site/src/util/code-snippets-map.json
Comment (dayhaysoos) [inline site/src/util/code-snippets-map.json:1]: Hey @Yash282003, can you please exclude this file from the PR?

Comment (Yash282003) [reply site/src/util/code-snippets-map.json:1]: @dayhaysoos Sure...I will do that!

Comment (Yash282003) [reply site/src/util/code-snippets-map.json:1]: @dayhaysoos Done!!!

**Review by EbonyLouis — CHANGES_REQUESTED** [review]
Hey @Yash282003, 
We don't want the file to be pushed into your PR as deleted, like it is here. We want to make sure that it's not added into your PR at all. This issue happened on another PR, try following the resources listed and if it's unsuccessful lmk and I can help you.  

Another solution would be to create a new PR and only push the changes to the `footer.js` file. So make sure that the `code-snippets-map.json` file isn't selected when committing your changes.  
https://github.com/TBD54566975/developer.tbd.website/pull/813/files#r1345905393

**Pull request closed without merging**